### PR TITLE
Lps 140576 master

### PR DESCRIPTION
**Pull request by SamZiemer**
Hi @tinatian I've updated the tests with what we discussed.

### CODE REVIEW

**Comment by SamZiemer** [comment]
ci:test:sf

**Comment by SamZiemer** [comment]
ci:test:relevant

**Comment by SamZiemer** [comment]
Hi @tinatian I wanted to see if you may have some time to review this pull request soon. Thank you.

File: modules/apps/static/portal-configuration/portal-configuration-persistence-test/src/testIntegration/java/com/liferay/portal/configuration/persistence/upgrade/test/UpgradeConfigurationPidUpgradeTest.java [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+ */
+@RunWith(Arquillian.class)
+public class UpgradeConfigurationPidUpgradeTest {
+
+	@ClassRule
+	@Rule
+	public static final LiferayIntegrationTestRule liferayIntegrationTestRule =
+		new LiferayIntegrationTestRule();
+
+	public void createConfiguration(Character separator) throws Exception {
```
Comment (tinatian) [inline modules/apps/static/portal-configuration/portal-configuration-persistence-test/src/testIntegration/java/com/liferay/portal/configuration/persistence/upgrade/test/UpgradeConfigurationPidUpgradeTest.java:58]: @SamZiemer Please make this method private, it is only used in current class, it does not need to be public. Also, please check all your other methods, make them have the proper visibility.

File: modules/apps/static/portal-configuration/portal-configuration-persistence-test/src/testIntegration/java/com/liferay/portal/configuration/persistence/upgrade/test/UpgradeConfigurationPidUpgradeTest.java [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+		Dictionary<String, String> dictionary = HashMapDictionaryBuilder.put(
+			"felix.fileinstall.filename",
```
Comment (tinatian) [inline modules/apps/static/portal-configuration/portal-configuration-persistence-test/src/testIntegration/java/com/liferay/portal/configuration/persistence/upgrade/test/UpgradeConfigurationPidUpgradeTest.java:60]: @SamZiemer I can see all the tests are using configuration generated by this method, which means every configuration entry in the database will have "felix.fileinstall.filename" in their dictionary. While, for those configurations added from UI, you won't have that property stored in database. Please add tests to cover it.

**Comment by tinatian** [comment]
@SamZiemer Please see my comments and resend the pull to liferay-core-infra.

Thanks.
Tina.

**Pull request closed without merging**